packages/vuetify/src/components/VOtpInput/VOtpInput.ts

```diff
@@ -0,0 +1,348 @@
+// Styles
+import '../VTextField/VTextField.sass'
+import './VOtpInput.sass'
+
+// Extensions
+import VInput from '../VInput'
+import { VRow, VCol } from '../VGrid'
```

Comment:
We don't use the grid system in components.

Reply:
Got u @KaelWD.
Any further suggestions/recommendations/conventions I should consider while fixing this one?

TNX for your time BTW ;)

Reply:
Fixed on https://github.com/vuetifyjs/vuetify/pull/12761/commits/e22170f91f0335b5b1ebd82bd849d34397707102